pkgs/desktops/gnome/extensions/clipboard-indicator/default.nix

```diff
@@ -1,18 +1,23 @@
-{ lib, stdenv, fetchFromGitHub }:
+{ lib, stdenv, fetchFromGitHub, gettext, glib }:
 
 stdenv.mkDerivation rec {
   pname = "gnome-shell-extension-clipboard-indicator";
-  version = "37";
+  version = "38";
 
   src = fetchFromGitHub {
     owner = "Tudmotu";
     repo = "gnome-shell-extension-clipboard-indicator";
     rev = "v${version}";
-    sha256 = "0npxhaam2ra2b9zh2gk2q0n5snlhx6glz86m3jf8hz037w920k41";
+    sha256 = "FNrh3b6la2BuWCsriYP5gG0/KNbkFPuq/YTXTj0aJAI=";
   };
 
   uuid = "clipboard-indicator@tudmotu.com";
 
+  nativeBuildInputs = [
+    gettext
+    glib
+  ];
+
   installPhase = ''
```

Comment:
A makefile has been added so you can now replace the `installPhase` by 

```nix
makeFlags = [
  "INSTALLPATH=${placeholder "out"}/share/gnome-shell/extensions/${uuid}/"
];
```

Reply:
Thanks for the heads-up. :+1: 
Updated